docs/docsite/rst/guide_modulehelper.rst

```diff
@@ -0,0 +1,520 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+.. _ansible_collections.community.general.docsite.guide_modulehelper:
+
+Module Helper guide
+===================
+
+
+Introduction
+^^^^^^^^^^^^
+
+Writing a module for Ansible is largely described in existing documentation.
+However, a good part of that is boilerplate code that needs to be repeated every single time.
+That is where ``ModuleHelper`` comes to assistance: a lot of that boilerplate code is done.
+
+
+Quickstart
+""""""""""
+
+See the `example from Ansible documentation <https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html#creating-a-module>`_
```

Comment:
Unfortunately that section has no label, so we cannot reference it with RST syntax :(

Reply:
I shall test it again, but I was under the impression that was a "normal http link" not a reference to a document element.

Reply:
It *is* a normal HTTP link, but it better should be a `:ref:`. Unfortunately it can't because that section has no label.